xbmc/video/VideoDatabase.cpp

```diff
@@ -5577,11 +5854,22 @@ void CVideoDatabase::UpdateTables(int iVersion)
     }
     m_pDS->close();
   }
+
+  if (iVersion < 118)
+  {
+    // create type table
+    m_pDS->exec("CREATE TABLE type (type_id INTEGER PRIMARY KEY, name TEXT)");
+    InitializeTypeTable();
+
+    // create type_link table
+    m_pDS->exec("CREATE TABLE type_link (file_id INTEGER PRIMARY KEY, media_id INTEGER, media_type TEXT, type_id INTEGER)");
+    m_pDS->exec("INSERT INTO type_link SELECT idFile, idMovie, 'movie', 1 FROM movie");
+  }
 }
 
 int CVideoDatabase::GetSchemaVersion() const
 {
-  return 116;
+  return 118;
```

Comment:
Why the jump 116 to 118?

Reply:
6 and 8 are good numbers in my culture, 4 and 7 are bad ones. As this is still in development and there is no compatibility requirement, I just changed this number for convenience. :) I'll change this to the right one when this feature is completed and ready to merge.